libs/libxslt/Makefile

```diff
@@ -147,6 +157,12 @@ define Package/xsltproc/install
 		$(1)/usr/bin/
 endef
 
+define Host/Install
```

Comment:
Drop this entire section. Even if more stuff gets installed. it's perfectly acceptable for host builds.